doc/mgr/telemetry.rst

```diff
@@ -64,53 +64,151 @@ the way Ceph is used.
+------------------
+
+To allow the *telemetry* module to start sharing data::
+
+  ceph telemetry on
+
+Please note: Telemetry data is licensed under the Community Data License
+Agreement - Sharing - Version 1.0 (https://cdla.io/sharing-1-0/). Hence,
+telemetry module can be enabled only after you add '--license sharing-1-0' to
+the 'ceph telemetry on' command.
+
+Telemetry can be disabled at any time with::
+
+  ceph telemetry off
+
 Sample report
 -------------
 
 You can look at what data is reported at any time with the command::
 
   ceph telemetry show
 
+If telemetry is off, you can preview a sample report with::
+
+  ceph telemetry preview
+
 To protect your privacy, device reports are generated separately, and data such
 as hostname and device serial number is anonymized. The device telemetry is
 sent to a different endpoint and does not associate the device data with a
 particular cluster. To see a preview of the device report use the command::
 
   ceph telemetry show-device
 
+If telemetry is off you can preview a sample device report with::
+
+  ceph telemetry preview-device
+
 Please note: In order to generate the device report we use Smartmontools
 version 7.0 and up, which supports JSON output. 
 If you have any concerns about privacy with regard to the information included in
 this report, please contact the Ceph developers.
 
-Channels
---------
+In case you prefer to have a single output of both reports, and telemetry is on, use::
 
-Individual channels can be enabled or disabled with::
+  ceph telemetry show-all
 
-  ceph config set mgr mgr/telemetry/channel_ident false
-  ceph config set mgr mgr/telemetry/channel_basic false
-  ceph config set mgr mgr/telemetry/channel_crash false
-  ceph config set mgr mgr/telemetry/channel_device false
-  ceph config set mgr mgr/telemetry/channel_perf false
-  ceph telemetry show
-  ceph telemetry show-device
+Otherwise use::
 
-Enabling Telemetry
-------------------
+  ceph telemetry preview-all
 
-To allow the *telemetry* module to start sharing data::
+**Sample report by channel**
 
-  ceph telemetry on
+When telemetry is on you can see what data is reported by channel with::
 
-Please note: Telemetry data is licensed under the Community Data License
-Agreement - Sharing - Version 1.0 (https://cdla.io/sharing-1-0/). Hence,
-telemetry module can be enabled only after you add '--license sharing-1-0' to
-the 'ceph telemetry on' command.
+  ceph telemetry show <channel_name>
 
-Telemetry can be disabled at any time with::
+Please note: If telemetry is on, and <channel_name> is disabled, the command
+above will output a sample report by that channel, according to the collections
+the user is enrolled to. However this data is not reported, since the channel
+is disabled.
 
-  ceph telemetry off
+If telemetry is off you can preview a sample report by channel with::
+
+  ceph telemetry preview <channel_name>
+
+Collections
+-----------
+
+Collections represent what we collect by channel.
```

Comment:
perhaps add something like 'different aspects of data that we collect within a channel'